### PR TITLE
Adds a Synthetic Residence Card

### PR DESCRIPTION
**Pull request by TheStryker**
In keeping with lore developments, this adds a new Synthetic Residence Card available in the accessories tab of the loadout.
image

https://forums.aurorastation.org/topic/2662-mendell-city-bugle/page/12/?tab=comments#comment-149380

### CODE REVIEW

**Review by Casper3667 — CHANGES_REQUESTED** [review]
From what I can see, you need to merge master into this branch.

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -411,6 +411,12 @@
 	TCFLcard["veteran"] = /obj/item/clothing/accessory/badge/tcfl_papers/service/veteran
 	gear_tweaks += new /datum/gear_tweak/path(TCFLcard)
 
+/datum/gear/accessory/syntheticcard
```
Comment (Casper3667) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm:414]: Given it is for IPC, it should probably be over in `loadout/loadout_xeno/machine.dm` instead and be whitelist restricted.
`whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)`

Comment (TheStryker) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm]: Thanks for the heads up, I was wondering how to whitelist a loadout item.

**Comment by TheStryker** [comment]
!review

**Review by Casper3667 — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Pull request merged**